ev3dev/core.py

```diff
@@ -2362,6 +2362,17 @@ def raw(self):
         return self.value(0), self.value(1), self.value(2)
 
     @property
+    def rgb(self):
+        """
+        Same as raw() but RGB values are scaled to 0-255
+        """
+        (red, green, blue) = self.raw
+
+        return (int((red * 255) / 1020),
```

Comment:
Why integers? It seems like we should return floats given that the sensor theoretically returns 4x the precision as can be represented in this range.

(yes, this is old code, but I'd like to take the chance to make improvements)

Reply:
Displaying RGB values in html or passing them to a lot of color libraries need them to be integers.

Reply:
If the sensor's accuracy is low enough that 1/4th the resolution isn't a detriment, I have no problem with this; otherwise, I think we either need to heavily document the behavior or add a new accessor/method.

Reply:
The accuracy of mine is pretty horrible, I think we are ok dropping the 1/4th resolution

Reply:
That certainly sounds familiar 😁 These LEGO sensors certainly aren't very accurate.

Reply:
It would probably be useful to normalize the values here as well. For example, the blue is always significantly less than red and green. Also, you will never get anywhere near 1020, so the scaled values currently will never be anywhere near 255.

Related discussion: https://www.facebook.com/groups/legomindstorms/permalink/662009207280642/

Reply:
Brainstorming....we could add a `calibrate_to_white()` method where we note the RGB values and then `rgb()` could scale from 0-255 based on the RGB values we stored when `calibrate_to_white()` ran.

Reply:
That video is interesting, I don't get colors anywhere close to that bright.  Let me flip back to the stock ev3 software and try calibrating my sensor.